search/string_utils.cpp

```diff
@@ -0,0 +1,26 @@
+#include "search/string_utils.hpp"
+
+#include "indexer/search_string_utils.hpp"
+
+namespace search
+{
+
+QueryString MakeQueryString(std::string s)
+{
+  QueryString qs;
+  qs.m_query = std::move(s);
```

Comment:
nit: `const & s` approach is safer, because you can't use moved `s` after this line later.

Reply:
Does the compiler automatically optimize this case with the `std::string s` argument?

```
std::string s = "a very long test string";
MakeQueryString(std::move(s));
```

This one is optimized, right?
```
MakeQueryString("a very long test string");
```

Reply:
This is the optimal way if you initialize a var from an input string.